mars/dataframe/base/to_numeric.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import pandas as pd
+from ..operands import DataFrameOperand, DataFrameOperandMixin
+from ...serialization.serializables import StringField
+from ..initializer import Series as asseries
+from ...tensor import tensor as astensor
+from ..core import SERIES_TYPE
+from ...tensor.core import TENSOR_TYPE, TensorOrder
+from ...core import ENTITY_TYPE, OutputType
+
+class DataFrameToNumeric(DataFrameOperand, DataFrameOperandMixin):
+
+    _errors = StringField("errors")
+    _downcast = StringField("downcast")
+
+    def __init__(self, errors="raise", downcast=None,**kw):
+        super().__init__(_errors=errors, _downcast=downcast,**kw)
+
+    @property
+    def errors(self):
+        return self._errors
+
+    @property
+    def downcast(self):
+        return self._downcast
```

Comment:
We are not encouraging using underline prefixes for fields any more. You may take a look at `TensorIsIn` operand.